sst/weatherForecast/Weather_Forecast.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>Suggestion: consider using <a href="https://pycqa.github.io/isort/" target="_blank">isort</a> in your notebook. This will reorganize your imports to comply with the <a href="https://peps.python.org/pep-0008/#imports" target="_blank">Imports guidelines in PEP-8</a>. </p><p> </p><p>In addition, move any project specific import to the end of the import list. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/reports-performance-summary/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done. Acutally I had to run <code>nbqa isort Weather_Forecast.ipynb</code>  for notebook</p>